lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebar.test.tsx

```diff
@@ -5,98 +5,101 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { ShallowWrapper } from 'enzyme';
-import 'jest-enzyme';
-import { faCrow, faBiohazard } from '@fortawesome/free-solid-svg-icons';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxGlobalSidebar, { Props } from '../NxGlobalSidebar';
-import NxButton from '../../NxButton/NxButton';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { faBiohazard, faCrow } from '@fortawesome/free-solid-svg-icons';
+import { rtlRender, runTimers, userEvent } from '../../../__testutils__/rtlUtils';
+import { screen } from '@testing-library/react';
 
 describe('NxGlobalSidebar', function() {
-  let minimalProps: Props,
-      toggleOpenSpy: () => void,
-      getShallowComponent: (optionalProps?: Partial<Props>) => ShallowWrapper;
-
-  beforeEach(function() {
-    toggleOpenSpy = jest.fn();
-    minimalProps = {
-      isOpen: false,
-      toggleOpenIcon: faCrow,
-      toggleCloseIcon: faBiohazard,
-      onToggleClick: toggleOpenSpy,
-      logoImg: 'logoImg',
-      logoAltText: 'alt text',
-      logoLink: '/ref'
-    };
-    getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxGlobalSidebar, minimalProps);
-  });
+  const minimalProps: Props = {
+    isOpen: false,
+    toggleOpenIcon: faCrow,
+    toggleCloseIcon: faBiohazard,
+    onToggleClick: jest.fn,
+    logoImg: 'logoImg',
+    logoAltText: 'alt text',
+    logoLink: '/ref'
+  };
 
-  it('renders an aside with nx-global-sidebar class', function() {
-    expect(getShallowComponent()).toMatchSelector('aside.nx-global-sidebar');
-  });
+  const quickRender = rtlRender<Props>(NxGlobalSidebar, minimalProps);
 
-  it('adds classes based on isOpen state', function() {
-    expect(getShallowComponent({ isOpen: true })).toMatchSelector('.nx-global-sidebar.open');
-    expect(getShallowComponent({ isOpen: false })).toMatchSelector('.nx-global-sidebar.closed');
+  it('renders a top-level element with role="complementary"', function() {
+    expect(quickRender().getByRole('complementary')).toBeInTheDocument();
   });
 
   it('renders a link to `logoLink` with the `logoImg` and the `altLogoText`', function() {
-    const component = getShallowComponent(),
-        link = component.find('.nx-global-sidebar__product-info'),
-        logo = link.find('.nx-global-sidebar__logo');
+    const view = quickRender(),
+        link = view.getByRole('link'),
+        logo = view.getByRole('img');
 
-    expect(link).toHaveProp('href', '/ref');
-    expect(logo).toHaveProp('src', 'logoImg');
-    expect(logo).toHaveProp('alt', 'alt text');
+    expect(link).toHaveAttribute('href', '/ref');
+    expect(logo).toHaveAttribute('src', 'logoImg');
+    expect(logo).toHaveAttribute('alt', 'alt text');
```

Comment:
I wonder if it would be better to check this as the accessible name of the image rather than specifically the alt text

Reply:
> I wonder if it would be better to check this as the accessible name of the image rather than specifically the alt text

That's a good idea. Changed here: 4944f73634008b984e99e18cb855427dd94c1e2d